lib/srv/reexec.go

```diff
@@ -275,6 +278,13 @@ func RunCommand() (errw io.Writer, code int, err error) {
 	var pty *os.File
 	uaccEnabled := false
 
+	localUser, err := user.Lookup(c.Login)
+	if err != nil {
+		// Ignore error since there's no way to log it.
+		_ = uacc.LogFailedLogin(c.UaccMetadata.BtmpPath, c.Login, c.UaccMetadata.Hostname, c.UaccMetadata.RemoteAddr)
```

Comment:
Why can't we log it? There are logging calls above, e.g.: `log.WithError(err).Debugf("failed to send UserEnd event to auditd: %v", err)`